application/src/main/java/org/togetherjava/tjbot/features/reminder/Reminder.java

```diff
@@ -0,0 +1,12 @@
+package org.togetherjava.tjbot.features.reminder;
+
+import org.togetherjava.tjbot.db.generated.tables.records.PendingRemindersRecord;
+
+public record Reminder(PendingRemindersRecord pendingReminders) {
+
+    public static Reminder from(PendingRemindersRecord pendingReminders) {
+        return new Reminder(pendingReminders);
+    }
```

Comment:
makes no sense to have a `from` method, just do `new Reminder(pendingReminder)` y the name is plural btw?

Reply:
then u need to create a second constructor that chains to the original per-value constructor? also possible